appium/webdriver/flutter_finder.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Tuple
+
+from appium.webdriver.common.appiumby import AppiumBy
+
+
+class FlutterFinder:
```

Comment:
I think this class would be nice to be in `appium/webdriver/extensions/flutter_integration`... since this is only for the flutter integration driver specific

Reply:
Thanks! updated